src/components/ProfileMatch/styles.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+export const IconDiv = styled.div`
+  min-width: 16px;
+  min-height: 16px;
+  text-align: center;
+  align-items: center;
+`;
```

Comment:
we can use `Box` for this (feel free to add a `text-align` prop if it's not there already)
also not sure if `align-items` does anything if it's not a flex or grid

Reply:
ok, i've changed it to Box and added text-align.